src/utils/getDependencyVersion.ts

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const parsePackageFile = async () => {
+  const packageJson = path.join(process.cwd(), './package.json');
+  const content = await fs.readFile(path.resolve(packageJson), 'utf-8', (err: Error, data: string) => {
```

Comment:
we can use readFileSync and reduce some of the complexity of the function

Reply:
agree